### PR TITLE
Issue#103-Adding logging to apis codes part2

### PR DESCRIPTION
**Pull request by juniartisu**

### CODE REVIEW

File: src/main/java/application/rest/v1/ComponentInfoRegistry.java [hunk 1]
```diff
@@ -284,16 +287,16 @@ private void processGroupVersion(ApiClient client, Map<String,ComponentInfo> map
                             String kind = resource.get("kind").getAsString();
                             String key = (group != null ? group : "") + "/" + version + "/" + kind;
                             String groupKindKey = (group != null ? group : "") + "/" + kind;
-                            System.out.println("processGroupVersion ComponentInfo map key: " + key);
+                            Logger.log(className, "processGroupVersion", Logger.LogType.INFO, "ComponentInfo map key: " + key);
```
Comment (rlcundiff) [inline src/main/java/application/rest/v1/ComponentInfoRegistry.java]: This group is INFO messages but I'm not sure why.  They look more like debug.

File: src/main/java/application/rest/v1/ComponentInfoRegistry.java [hunk 2]
```diff
@@ -444,18 +453,23 @@ public abstract Object getNamespacedObject(CoreV1Api api, ComponentInfo info, St
         @Override
         public Object listClusterObject(CoreV1Api api, ComponentInfo info, String pretty, String labelSelector,
                 String resourceVersion, Boolean watch) throws ApiException {
+            Logger.log(ConfigMapResolver.class.getName(), "listClusterObject", Logger.LogType.INFO, "For componentInfo=" + info
```
Comment (rlcundiff) [inline src/main/java/application/rest/v1/ComponentInfoRegistry.java]: Serveral more here that are INFO.  Wouldn't DEBUG make more sense?

File: src/main/java/application/rest/v1/KAppNavEndpoint.java [hunk 3]
```diff
@@ -443,25 +468,29 @@ private static String getErrorMessageAsJSON(String msg) {
     // Generic CRUD APIs 
 
     protected Object getNamespacedGenericObject(ApiClient client, String group, String kindPlural, String namespace, String name) throws ApiException {
+        Logger.log(className, "getNamespacedGenericObject", Logger.LogType.INFO, "For group=" + group + ", kindPlural="+kindPlural + ", namespace="+namespace + ", name="+name);
```
Comment (rlcundiff) [inline src/main/java/application/rest/v1/KAppNavEndpoint.java]: Another set of INFOs that look like they should be DEBUG.

File: src/main/java/com/ibm/kappnav/logging/Logger.java [hunk 4]
```diff
@@ -66,33 +66,38 @@ public static void setLogLevel(String level) {
...
+                    enabled[LogType.ENTRY.ordinal()]= true;  
+                    enabled[LogType.EXIT.ordinal()]= true;  
+                    break;
+                case "debug": 
+                    setLogTypes(true); 
+                    break;
+                case "all": 
+                    setLogTypes(true); 
+                    break;
+                default : // same as info
```
Comment (rlcundiff) [inline src/main/java/com/ibm/kappnav/logging/Logger.java:96]: Could add a debug here noting the invalid level value requested.

**Review by rlcundiff — CHANGES_REQUESTED** [review]

File: src/main/java/application/rest/v1/ComponentInfoRegistry.java (hunk 1 above)
Comment (juniartisu) [reply src/main/java/application/rest/v1/ComponentInfoRegistry.java]: I changed it to use DEBUG.

File: src/main/java/application/rest/v1/ComponentInfoRegistry.java (hunk 2 above)
Comment (juniartisu) [reply src/main/java/application/rest/v1/ComponentInfoRegistry.java]: I changed it to use DEBUG.

File: src/main/java/application/rest/v1/KAppNavEndpoint.java (hunk 3 above)
Comment (juniartisu) [reply src/main/java/application/rest/v1/KAppNavEndpoint.java]: I changed it to use DEBUG.

File: src/main/java/com/ibm/kappnav/logging/Logger.java (hunk 4 above)
Comment (juniartisu) [reply src/main/java/com/ibm/kappnav/logging/Logger.java:96]: Adding debug msg.

**Review by rlcundiff — APPROVED** [review]

**Pull request merged**